pkgs/development/python-modules/tensorflow/default.nix

```diff
@@ -104,27 +120,22 @@ let
 
   bazel-build = buildBazelPackage {
     name = "${pname}-${version}";
-    bazel = bazel_3;
 
     src = fetchFromGitHub {
       owner = "tensorflow";
       repo = "tensorflow";
       rev = "v${version}";
-      sha256 = "07a2y05hixch1bjag5pzw3p1m7bdj3bq4gdvmsfk2xraz49b1pi8";
+      sha256 = "sha256:0736mx2b9q8fzyl523h1qlqmjqbngx36c9fmddzavw09az49bhh7";
     };
 
-    patches = [
-      # included from 2.6.0 onwards
-      (fetchpatch {
-        name = "fix-numpy-1.20-notimplementederror.patch";
-        url = "https://github.com/tensorflow/tensorflow/commit/b258941525f496763d4277045b6513c815720e3a.patch";
-        sha256 = "19f9bzrcfsynk11s2hqvscin5c65zf7r6g3nb10jnimw79vafiry";
-      })
-      # Relax too strict Python packages versions dependencies.
-      ./relax-dependencies.patch
-      # Add missing `io_bazel_rules_docker` dependency.
-      ./workspace.patch
-    ];
+    # Relax too strict Python packages versions dependencies.
+    patchPhase = ''
+      sed -i 's/typing_extensions ~= 3.7.4/typing_extensions >= 3.7.4/' tensorflow/tools/pip_package/setup.py
```

Comment:
Please use substituteInPlace instead.

Reply:
Done in https://github.com/NixOS/nixpkgs/pull/137069/commits/228e8203c3b07d8f2351c73bc6af90f6fb73cc38.